packages/gitgraph-react/src/GitgraphReact.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from "react";
+import { GitGraph as GitGraphCore, GitGraphOptions } from "gitgraph-core/lib/index";
```

Comment:
I don't yet if it's a good idea, but I was thinking about renaming `GitGraph` to `GitgraphCore` to avoid this kind of pattern and also permit to consolidate every types (I want to be able to load `Branch`, `Commit`… from `gitgraph-react` directly)